app/src/ui/repository-settings/repository-settings.tsx

```diff
@@ -198,6 +233,19 @@ export class RepositorySettings extends React.Component<
       }
     }
 
+    if (
```

Comment:
took me a minute to read this `if` statement correctly
```suggestion
    // only update this if it will be different from what we have stored
    if (
```